src/main/java/com/spotinst/sdkjava/model/SpotinstRepoManager.java

```diff
@@ -28,6 +28,7 @@
     private ISpotAccountAdminRepo                   spotAdminAccountRepo;
     private ISpotOceanEcsLaunchSpecRepo             spotOceanEcsLaunchSpecRepo;
     private ISpotStorageAzureVolumeRepo             spotStorageAzureVolumeRepo;
+    private ISpotOceanK8sVirtualNodeGroupRepo       spotOceanK8sVirtualNodeGroupRepo;
```

Comment:
Classname will change when VirtualNodeGroup is changed to 'Vng' throughout the PR.